src/rgw/rgw_lc.cc

```diff
@@ -1617,7 +1617,7 @@ static inline vector<int> random_sequence(uint32_t n)
     [ix = 0]() mutable {
       return ix++;
     });
-  std::random_shuffle(v.begin(), v.end());
+  std::shuffle(v.begin(), v.end(), std::mt19937(std::random_device()()));
```

Comment:
```suggestion
  std::random_device rd;
  std::default_random_engine rng{rd()};
  std::shuffle(ranks.begin(), ranks.end(), rng);
```

i'd suggest break this one liner into smaller clauses for better readability.

Reply:
@tchaikov 
I'll do that, but the single liner was sort of charming.